### PR TITLE
chore: Prepare for 0.3.0

### PR DESCRIPTION
**Pull request by jparsai**
Signed-off-by: jparsai <jparsai@redhat.com>

### CODE REVIEW

**Comment by jparsai** [comment]
Hello @jeffmaury PTAL at this PR to release 0.3.0 version version of Dependency Analytics plugin.

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -87,6 +87,9 @@
   ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>0.3.0</b></p>
+  <p>Stack Analysis capability is implementation to see detailed analysis of stack.</p>
```
Comment (jeffmaury) [inline src/main/resources/META-INF/plugin.xml:91]: Shouldn't be 'is implemented ...' ?

Comment (jparsai) [reply src/main/resources/META-INF/plugin.xml:91]: Ohh Sorry I was writing "Implementation of SA capabilities" then I changed it, and missed it.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**